modules/reference/pages/configuration-parameters.adoc

```diff
@@ -1234,20 +1234,20 @@ used to compress equals the node CPU’s cores. |`0`
 
 |System.Backup.Local.Enable |Backup data to local path *IMPORTANT*: If set to `true`, this also enables a daily full backup at 12:00am UTC. |`false`
 
-|System.Backup.Local.Path |The path to store the backup files |`nan`
-|System.Backup.S3.AWSAccessKeyID |The AWS access key ID for s3 bucket of
-backup |`nan`
+|System.Backup.Local.Path |The path to store the backup files |`null`
+|System.Backup.S3.AWSAccessKeyID |The AWS access key ID used for the S3 backup bucket. It corresponds to the "Access Key" in GCS and the "Account Name" in Azure Blob, respectively. | `null`
 
-|System.Backup.S3.AWSSecretAccessKey |The secret access key for s3
-bucket |`nan`
+|System.Backup.S3.AWSSecretAccessKey |The secret access key used for the AWS S3 bucket. It corresponds to the "Secret" in GCS and the "Account Key" in Azure Blob, respectively.  |`null`
 
-|System.Backup.S3.BucketName |The S3 bucket name |`nan`
```

Comment:
unify using `null` to represent the default empty string ""  for the backup string configuration item.